example.env

```diff
@@ -1,9 +0,0 @@
-FRONTEND_PORT=80
-BACKEND_PORT=8080
-DATABASE_PORT=5432
-POSTGRES_USER=root
-POSTGRES_PASSWORD=pass
-POSTGRES_DB=data
-
-NEXT_PUBLIC_FRONTEND_URI=http://localhost:80
-NEXT_PUBLIC_BACKEND_URI=http://localhost:8080
```

Comment:
Is there a reason we are deleting the example.env?

Reply:
I did not actually delete it I renamed it to .env and thats in gitignore

Reply:
Okay, we should still leave this even when .env exists. This is meant to support the documentation and provide a mock template for creating the actual .env file. So I'd ask for you to keep your local .env as it is now, and just create this file again

Reply:
Oke